backend/apps/github/migrations/0045_add_calculated_score.py

```
@@ -0,0 +1,15 @@
+        ("github", "0044_user_indexes"),
+    ]
+    operations = [
+        migrations.AddField(
+            model_name="user",
+            name="calculated_score",
+            field=models.FloatField(
+                default=0,
+                help_text="Computed score based on contribution signals",
+            ),
+        ),
+    ]
```

Ruff (0.15.10)

[warning] No newline at end of file

Add trailing newline

(W292)